fern/patches/ghostery-branding.js

```diff
@@ -3,41 +3,49 @@ const path = require("path");
 const fsExtra = require("fs-extra");
 
 const { getRoot } = require("../core/workspace.js");
-const { folderExists } = require("../core/utils.js");
 
 async function getPathToSourceBranding() {
   const root = await getRoot();
   return path.join(root, "brands", "ghostery", "branding");
 }
 
+async function getPathToSourceWindowsInstallerIcon() {
+  const root = await getRoot();
+  return path.join(root, "brands", "ghostery", "windows-installer", "setup.ico");
+}
+
 async function getPathToGhosteryBranding() {
   const root = await getRoot();
   return path.join(root, "mozilla-release", "browser", "branding", "ghostery");
 }
 
+const windowsInstallerIconPathComponents = [
+  "other-licenses",
+  "7zstub",
+  "firefox",
+  "setup.ico",
+];
+
 async function getPathToWindowsInstallerIcon() {
   return path.join(
     await getRoot(),
     "mozilla-release",
-    "other-licenses",
-    "7zstub",
-    "firefox",
-    "setup.ico"
+    ...windowsInstallerIconPathComponents
   );
 }
 
 module.exports = {
   name: "Setup Ghostery branding",
-  paths: ["browser/branding/ghostery"],
-  skip: async () => folderExists(await getPathToGhosteryBranding()),
+  paths: ["browser/branding/ghostery", path.join(...windowsInstallerIconPathComponents)],
+  skip: async () => false,
```

Comment:
If it already exists, will `fsExtra.copy` succeed? (I would expect to have an exception raised?

Reply:
Does not complain to me